### PR TITLE
Optional label

### PR DESCRIPTION
**Pull request by ethen001**
### Issues
#73 

- [x] make label optional
- [x] fixed unit test for post form such that they cover new changes
### Technical Changes
* added variable to `PostForm`
* modified submit form of `PostForm`
* made input `label` optional
* added variable to `EditPostForm`

### Help!
I couldn't find an acceptance criteria for these changes. Thus, I have a couple of questions. Is this what needs to be considered for the label optonal? Do you like how I implemented the label optional? Would you prefer if I make it as 3 dadio buttons: Nono, General, University, instead?

### CODE REVIEW

**Comment by songrenzhao** [comment]
I think having 2 radio buttons is good. 
One for general and second one for university @ethen001

**Comment by uting98** [comment]
I think the way you did Label optional is pretty good, no need to add radio button, since it is optional, we are not forcing user to choose one label to be present

File: src/components/Forms/PostForm/PostForm.js [hunk 1]
```diff
@@ -37,8 +39,8 @@ export const PostForm = ({
 
   const onSubmit = (e) => {
     e.preventDefault();
-    if (!formData.label.trim() || !formData.title.trim() || !formData.content.trim()) {
-      setForm({ label: 'general', title: '', content: '' });
+    if (!formData.title.trim() || !formData.content.trim()) {
+      setForm({ label: '', title: '', content: '' });
```
Comment (JiejayLan) [inline src/components/Forms/PostForm/PostForm.js]: If one of the necessary field is empty, you should not clear all input. Otherwise

Comment (ethen001) [reply src/components/Forms/PostForm/PostForm.js]: I agree

**Pull request merged**